### DISCUSSION TITLE
folder level .pages nav not working when top level .pages nav has sub-sections?

### DISCUSSION THREAD

**Discussion by paulinestacey** [squidfunk/mkdocs-material#5885] [Q&A] [answered]
My docs project has a top level .page nav which I've edited to show some sub-sections (at the folder level). The top nav now renders exactly as we want it, but the .pages nav files within the folders are being ignored so the docs files are rendering in default order with default titles. If I reinstate the original top level .page nav without sub-sections, all the .pages navs in the project render correctly.
Any one else had this? got suggestions to fix? 
Many thanks :) 

(PS. running Mkdocs 1.5.2, Material 9.1.21, awesome-pages-plugin 2.8.0)

**Comment by squidfunk** [comment, marked as answer]
Please report this to awesome pages.

**Reply by paulinestacey** [reply]
Thanks for quick response. Will do.

**Reply by paulinestacey** [reply]
In case anyone else has this issue, upgrading awesome-pages plugin to the latest v2.9.2 fixed this issue.
